modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -152,7 +152,11 @@ def _dt_prop_map(property_name):
 
     def dt_op_builder(df, *args, **kwargs):
         """Access specified date-time property of the passed frame."""
-        prop_val = getattr(df.squeeze(axis=1).dt, property_name)
+        squeezed_df = df.squeeze(axis=1)
+        if isinstance(squeezed_df, pandas.DataFrame) and len(squeezed_df.columns) == 0:
+            return pandas.DataFrame([])
```

Comment:
```suggestion
            return squeezed_df
```

Should we return `squeezed_df` so we get the index, too?